app/controllers/distributions_controller.rb

```diff
@@ -47,10 +47,17 @@ def index
     @total_items_paginated_distributions = total_items(@paginated_distributions)
     @items = current_organization.items.alphabetized
     @partners = @distributions.collect(&:partner).uniq.sort_by(&:name)
-    @statuses = Distribution.states.transform_keys(&:humanize)
     @selected_item = filter_params[:by_item_id]
     @selected_partner = filter_params[:by_partner]
     @selected_status = filter_params[:by_state]
+    # FIXME: one of these needs to be removed but it's unclear which at this point
+    @statuses = Distribution.states.transform_keys(&:humanize)
+    @states = Distribution.states.transform_keys(&:humanize)
```

Comment:
Why do we have `@statues` and `@states` that appear to be identical?

Reply:
It was unclear which one was necessary. I can suss it out.

Reply:
Done! `@states` gets the axe!